src/views/Swap/Swap.vue

```diff
@@ -587,24 +598,25 @@ export default {
     availableAmount () {
       return dpUI(this.available, VALUE_DECIMALS)
     },
-    ethRequired () {
-      if (this.assetChain === 'ETH') {
-        return !this.account?.balances?.ETH || this.account?.balances?.ETH === 0
+    noTokens () {
+      if (this.networkWalletBalances[this.asset] < this.min) {
+        return true
       }
-
-      if (this.toAssetChain === 'ETH') {
-        return !this.toAccount?.balances?.ETH || this.toAccount?.balances?.ETH === 0
+      if (this.networkWalletBalances[this.toAsset] < this.min) {
```

Comment:
what does the `toAsset` have to do with the minimum ?
minimum is always denominated in the from asset